pkg/controllers/core/catalog_controller_test.go

```diff
@@ -48,6 +49,12 @@ var _ = Describe("Catalogd Controller Test", func() {
 	)
 	BeforeEach(func() {
 		ctx = context.Background()
+		if err := features.CatalogdFeatureGate.Set("PackagesBundleMetadataAPIs=true"); err != nil {
+			Expect(err).NotTo(HaveOccurred())
+		}
+		if err := features.CatalogdFeatureGate.Set("CatalogMetadataAPI=true"); err != nil {
+			Expect(err).NotTo(HaveOccurred())
+		}
```

Comment:
Nit: 
```suggestion
Expect(features.CatalogdFeatureGate.Set("PackagesBundleMetadataAPIs=true")).NotTo(HaveOccurred())
Expect(features.CatalogdFeatureGate.Set("CatalogMetadataAPI=true")).NotTo(HaveOccurred())
```

IMO this is more succinct and easier to read